packages/ketcher-react/src/script/ui/views/components/ContextMenu/items/FunctionalGroupBatchOperations.tsx

```diff
@@ -0,0 +1,125 @@
+import { Action, fromSgroupDeletion, setExpandSGroup } from 'ketcher-core'
+import { useCallback } from 'react'
+import { Item, PredicateParams } from 'react-contexify'
+import { useDispatch } from 'react-redux'
+import { useAppContext } from 'src/hooks'
+import Editor from 'src/script/editor'
+import { highlightFG } from 'src/script/ui/state/functionalGroups'
+import {
+  ContextMenuShowProps,
+  CustomItemProps,
+  ItemData
+} from '../contextMenu.types'
+
+const FunctionalGroupBatchOperations: React.FC<CustomItemProps> = (
+  properties
+) => {
+  const { getKetcherInstance } = useAppContext()
+  const dispatch = useDispatch()
+
+  const isExpandHidden = useCallback(
+    ({
+      props,
+      triggerEvent,
+      data
+    }: PredicateParams<ContextMenuShowProps, ItemData>) => {
+      if (properties.hidden({ props, triggerEvent, data })) {
+        return true
+      }
+
+      return Boolean(
```

Comment:
Minor: `every` method itself returns a Boolean

Reply:
Thank you, but it will return `undefined` if `props?.functionalGroups?` is `undefined`